src/Sulu/Bundle/RouteBundle/Generator/GeneratedRoute.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RouteBundle\Generator;
+
+use Sulu\Bundle\RouteBundle\Model\RoutableInterface;
+
+/**
+ * Represents a newly generated route. It contains additional the used entity and class to generate the route.
+ */
+class GeneratedRoute
```

Comment:
@danrot your right the persisting of the entity is done somewhere else so we could simple instantiating it.